python/ray/serve/scripts.py

```diff
@@ -763,6 +764,7 @@ def build_app_config(import_path: str, name: str = None):
             "port": DEFAULT_GRPC_PORT,
             "grpc_servicer_functions": grpc_servicer_functions,
         },
+        "logging_config": {**LoggingConfig().dict()},
```

Comment:
Non-blocker: isn't `LoggingConfig().dict()` already returns a dictionary? Is it necessary to case to unpack to a new dict by doing `{**xyz}`?

Reply:
done! good catch.